test/functional/fixtures/reporter/testcafe-fixtures/report-data-test.js

```diff
@@ -0,0 +1,8 @@
+fixture`Report Data API`
+    .page('../pages/index.html');
+
+test('Run t.report action with object val', async t => {
```

Comment:
I propose to use some general name: `test`, `Run t.report action` or something else.